index.html

```diff
@@ -20,7 +20,7 @@
 				<div class="nav">
 					<div class="nav__social-bar">
 						<a href="#" class="logo">
-							<h4 class="logo__h4">AURORA.</h4>
+							<h4 class="logo__h4">Aurora.</h4>

```

Comment:
```suggestion
							<h4 class="logo__h4">Aurora</h4>

```
pls remove unused code